pkg/api/authz.go

```diff
@@ -228,6 +228,16 @@ func isBearerAuthEnabled(config *Config) bool {
 	return false
 }
 
+func isAuthnEnabled(config *Config) bool {
+	if config.AccessControl != nil &&
```

Comment:
Remove this access control "if"

Reply:
Ok, I didn't understand in the first place. I also moved isAuthnEnabled and isBearerAuthEnabled in auth.go.